pkg/systemd/logs.jsx

```diff
@@ -0,0 +1,395 @@
+                <Toolbar>
+                    <ToolbarContent>
+                        <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="lg">
+                            <ToolbarItem>
+                                <Select toggleId="logs-predefined-filters"
+                                        isOpen={isOpenTimeFilter}
+                                        onToggle={setIsOpenTimeFilter}
+                                        onSelect={(e, selection) => {
+                                            setIsOpenTimeFilter(false);
+                                            onTimeFilterChange(selection);
+                                        }}
+                                        selections={timeFilter}
+                                        placeholderText={_("Time")}>
+                                    {timeFilterOptions.map(option => <SelectOption key={option.value}
+                                                                                   value={option} />)}
+                                </Select>
+                            </ToolbarItem>
+
+                            <ToolbarItem variant="label">
+                                {_("Identifier")}
+                            </ToolbarItem>
+                            <ToolbarItem id="journal-identifier-menu">
+                                <IdentifiersFilter currentIdentifiers={currentIdentifiers}
+                                                onIdentifiersFilterChange={onIdentifiersFilterChange}
+                                                identifiersFilter={identifiersFilter} />
+                            </ToolbarItem>
+
+                            {showTextSearch &&
+                            <>
+                                <ToolbarItem variant="label">
+                                    {_("Filters")}
+                                </ToolbarItem>
+                                <ToolbarItem className="text-search">
+                                    <TextFilter id="journal-grep"
+                                                key={textFilter}
+                                                textFilter={textFilter} onTextFilterChange={onTextFilterChange} />
+                                </ToolbarItem>
+                                <ToolbarItem className="text-help">
+                                    <Popover id="logs-help-popover"
+                                             hasAutoWidth
+                                             bodyContent={
+                                                 <div className="logs-help-menu">
+                                                     <span className="help-links">
+                                                         <Button variant="link" component="a" isInline
+                                                                 href="https://www.freedesktop.org/software/systemd/man/journalctl.html"
```

Comment:
Did you deliberately drop the [RHEL documentation link](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/reviewing-logs_system-management-using-the-rhel-8-web-console#filtering-logs-in-the-web-console_reviewing-logshere?) I thought it was quite nice. Or does this not apply to the new page any more?

Reply:
Yes. The inline help explains all what is possible and I think it is even more clear now that through the rhel docs.